pkgs/_macro_host/test/macro_host_test.dart

```diff
@@ -62,9 +65,11 @@ void main() {
                     target: QualifiedName(
                         uri: 'package:foo/foo.dart', name: 'Foo'))),
             Scope.macro.run(() => AugmentResponse(augmentations: [
-                  Augmentation(
-                      code:
-                          '// {"uris":{"package:foo/foo.dart":{"scopes":{}}}}')
+                  Augmentation(code: [
+                    Code.resolvedCode(ResolvedCode(
+                        code:
+                            '// {"uris":{"package:foo/foo.dart":{"scopes":{}}}}'))
```

Comment:
I have... literally no idea what this is doing 🤣

Reply:
Looking at this with fresh eyes I see it is really just setting up an expected response.